Doppler.MercadoPagoApi/Controllers/PaymentController.cs

```diff
@@ -1,15 +1,29 @@
+using Doppler.MercadoPagoApi.DopplerSecurity;
+using Doppler.MercadoPagoApi.Models;
+using Doppler.MercadoPagoApi.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
 
 namespace Doppler.MercadoPagoApi.Controllers
 {
     [Authorize]
     [ApiController]
     public class PaymentController : Controller
     {
-        public PaymentController()
+        private readonly IMercadoPagoService _mercadoPagoService;
+
+        public PaymentController(IMercadoPagoService mercadoPagoService)
         {
+            _mercadoPagoService = mercadoPagoService;
+        }
 
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPost("/customers/{customername}")]
```

Comment:
Is [I have commented yesterday](https://github.com/FromDoppler/doppler-mercadopago-api/pull/4#discussion_r864743979), to use `OWN_RESOURCE_OR_SUPERUSER`, the URL should follow the convention `/accounts/{accountname}/`

Reply:
> Is [I have commented yesterday](https://github.com/FromDoppler/doppler-mercadopago-api/pull/4#discussion_r864743979), to use `OWN_RESOURCE_OR_SUPERUSER`, the URL should follow the convention `/accounts/{accountname}/`

But, does it has sense? the customername is the same than the accountname?

Reply:
In that case, using `OWN_RESOURCE_OR_SUPERUSER` with `/accounts/{accountname}/` sounds good because it guarantees that Doppler identifies the person with that email.